autopts/ptsprojects/zephyr/gap.py

```diff
@@ -223,6 +230,373 @@ def test_cases(ptses):
+                                         TestFunc(lambda: btp.gap_set_io_cap(IOCap.keyboard_display)),
+                                         TestFunc(lambda: btp.l2cap_listen(psm=0x1001, transport=defs.L2CAP_TRANSPORT_BREDR, mtu=br_init_mtu, response=L2CAPConnectionResponse.insufficient_authentication))
+                                         ],
+                  generic_wid_hdl=gap_wid_hdl),
+        ZTestCase("GAP", "GAP/SEC/SEM/BI-19-C",
+                  cmds=pre_conditions + [TestFunc(lambda: stack.l2cap_init(psm=0x1001, initial_mtu=br_init_mtu)),
+                                         TestFunc(lambda: btp.gap_set_io_cap(IOCap.keyboard_display)),
+                                         TestFunc(lambda: btp.l2cap_listen(psm=0x1001, transport=defs.L2CAP_TRANSPORT_BREDR, mtu=br_init_mtu, response=L2CAPConnectionResponse.insufficient_authentication))
+                                         ],
+                  generic_wid_hdl=gap_wid_hdl),
+        ZTestCase("GAP", "GAP/SEC/SEM/BI-04-C",
+                  cmds=pre_conditions + [TestFunc(lambda: stack.l2cap_init(psm=0x1001, initial_mtu=br_init_mtu)),
+                                         TestFunc(lambda: btp.gap_set_io_cap(IOCap.keyboard_display)),
+                                         TestFunc(lambda: btp.l2cap_listen(psm=0x1001, transport=defs.L2CAP_TRANSPORT_BREDR, mtu=br_init_mtu, response=L2CAPConnectionResponse.insufficient_authentication))
+                                         ],
+                  generic_wid_hdl=gap_wid_hdl),
+        ZTestCase("GAP", "GAP/SEC/SEM/BI-08-C",
+                  cmds=pre_conditions + [TestFunc(lambda: stack.l2cap_init(psm=0x1001, initial_mtu=br_init_mtu)),
+                                         TestFunc(lambda: btp.gap_set_io_cap(IOCap.keyboard_display)),
+                                         TestFunc(lambda: btp.l2cap_listen(psm=0x1001, transport=defs.L2CAP_TRANSPORT_BREDR, mtu=br_init_mtu, response=L2CAPConnectionResponse.insufficient_authentication))
+                                         ],
+                  generic_wid_hdl=gap_wid_hdl),
+        ZTestCase("GAP", "GAP/SEC/SEM/BI-31-C",
+                  cmds=pre_conditions + [TestFunc(lambda: stack.l2cap_init(psm=0x1001, initial_mtu=br_init_mtu)),
+                                         TestFunc(lambda: btp.gap_set_io_cap(IOCap.keyboard_display)),
+                                         TestFunc(lambda: btp.l2cap_listen(psm=0x1001, transport=defs.L2CAP_TRANSPORT_BREDR, mtu=br_init_mtu, response=L2CAPConnectionResponse.insufficient_authentication))
+                                         ],
+                  generic_wid_hdl=gap_wid_hdl),
+        ZTestCase("GAP", "GAP/DISC/LIMM/BV-01-C",
+                  cmds=pre_conditions + [TestFunc(gap_set_ad_flags, AdFlags.le_limit_discov_mode, iut_device_name), # limit discovery mode only
+                                         ],
+                  generic_wid_hdl=gap_wid_hdl),
```

Comment:
Do we have any style guides for autopts? The length of some of these lines does not conform to e.g. PEP8, but if autopts has not adopted/specified any style guides, then I guess it's fine